NSS-Likeable-Lemurs/BangazonSite
# Issue # 2

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/2)

## Description of Proposed Changes
-Changed get method for products on controller
-Changed html view for productType
-Changed html view for product Type list

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout <dsSetUp>
```

1.Press play.
2.Press on Product Type.
3. See List of all Categories.
4. Press on one of the Hyperlinks for the categories provided.
5. See list of all titles, Quantities, and price of all products within this category.

## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductTypesController
* ProductTypes Index.cshtml
* ProductTypes Details.cshtml

## Mentions @username

@HelenChalmers 
@ThraxHamilton 
@streatorward 
@kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
